externals/vjson/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project(VJSON)
+
+add_library(cvmfs_vjson STATIC src/block_allocator.cpp src/json.cpp)
+set_property(TARGET cvmfs_vjson PROPERTY POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
Is this using the `CVMFS_BASE_CXX_FLAGS` options?

Reply:
I will check that it does.